docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,43 @@
+# How to add self-signed certificate to trusted certificates list
+
+## Overview
+
+For secure connectivity to work you need to add CA certificate to the trusted certificate list repository.
+For the most part the store will be filled with certificates from your tls/ssl provider, but when you want to use self-signed certificates you may need to update your local certificate store.
```

Comment:
"the store" - is new terminology at this point - what store? I think "store" is confusing as it bring to mind Java keystore - which this is not.

`tls` and `ssl` should be capitalised.

"from your tls/ssl provider" - what does this mean?